drivers/console/semihosting_console.c

```diff
@@ -0,0 +1,125 @@
+/* semihosting_console.c - Console messages over semihosting */
```

Comment:
I don't know about the exact SPDX requirements, but most other Zephyr files have a comment with SPDX-License-Identifier: first in the file, with other comments following it.

Reply:
Right - I copied this over from another of the console drivers. Will fix